### ISSUE TITLE
"Incorrect" {callsite} value in log file

### ISSUE THREAD

**Issue by dulfe** [NLog/NLog.Owin.Logging#13]
Today I used this library for the first time and I when I was reviewing the log files it generated, I found all calling methods to be "`NLog.Owin.Logging.NLogFactory+Logger.WriteCore`" instead of my class methods (using `${callsite}`). Digging into the the code for _NLog_ and this _library_ I found out that NLog goes up the stack frame and finds the calling method, but since it does not "know" anything about your library, it assumes it is the correct one. (which is wrong in this case)

The problem gets solved by adding these 2 lines to my project, but I think it should be added to the library instead, so the end-user(developer) does not have to worry about it.

```
LogManager.AddHiddenAssembly(typeof(NLogFactory).Assembly);
LogManager.AddHiddenAssembly(typeof(LoggerExtensions).Assembly);
```

Hope it helps someone.

**Comment by 304NotModified** [comment]
Thanks for the report!

Is 

```
LogManager.AddHiddenAssembly(typeof(LoggerExtensions).Assembly);
```

really needed?

**Comment by 304NotModified** [comment]
Please let me know, then I will release a new version :)

**Comment by dulfe** [comment]
Yes, if you do not add it, the log files show LoggerExtensions as the calling source.

Thank you!

**Comment by 304NotModified** [comment]
From which package/library are those loggerextensions? Not this one?

**Comment by dulfe** [comment]
That is part of [Katana Owin Logging](http://katanaproject.codeplex.com/SourceControl/latest#src/Microsoft.Owin/Logging/LoggerExtensions.cs), that calls your logger, so it shows in the stack frame.

**Comment by 304NotModified** [comment]
Ow yes indeed. Thanks!

**Comment by 304NotModified** [comment]
Fixed in 1.0.1!
